docs/index.md

```diff
@@ -89,7 +89,7 @@ You can install Powertools for AWS Lambda (Python) using your favorite dependenc
             === "CDK"
 
                 ```python hl_lines="13 19"
-                --8<-- "examples/homepage/install/x86_64/cdk.py"
+                --8<-- "examples/homepage/install/x86_64/cdk_x86.py"
```

Comment:
renaming is probably a different pull request?

Reply:
Hey @Wurstnase! I think its ok keep it here, so we don't need to open a new PR for this.